src/atopile/cli/view.py

```diff
@@ -43,9 +43,10 @@ async def send_viewer_data():
 
 class Pose(BaseModel):
     """The position, orientation, flipping etc... of an element."""
-    x: float
-    y: float
-    angle: int  # degrees, but should only be 0, 90, 180, 270
+    position: dict[str, float] # {x: 0.0, y: 0.0}
```

Comment:
Not as strict enforcement as it was (since we don't know the contents of the dict). Hopefully that is fine?